nhais/generate_jsonschema.py

```diff
@@ -0,0 +1,26 @@
+import json
+from jsonschema import validate
+import os
+
+import utilities.integration_adaptors_logger as log
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+
+def validate_schema(request):
+    request_body = json.loads(request)
+
+    ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
We'll take a performance hit loading this file for every request. Maybe wrap this in class the load the schema(s) in the __init__?  That might not be enough though because a new instance of Handler is created for every class.

Reply:
Currently moved to handler init, but yes I see your point. Not sure how else we can have this loaded before the handler